src/main/java/net/praqma/tracey/cli/Tracey.java

```diff
@@ -38,6 +34,7 @@ public static void main(String[] args) throws Exception {
         sayParser.addArgument("-n", "--node").dest("node").help("The URL of the RabbitMQ server");
```

Comment:
@alexsedova When it comes to UI it is very important to always think about the user. Our users have no idea about how the tool works inside. They have no idea what default values will be used and when. This help screen is a natural way for users to discover how to use the tool - if it is bad they won't.

So, stop thinking as a developer and start to think as a user of the tool.

I would even add examples of configuration file to support users